### DISCUSSION TITLE
Im getting "ModuleNotFoundError: No module named 'optimizedSD'"

### DISCUSSION THREAD

**Discussion by LeandroLosaria** [basujindal/stable-diffusion#126] [Q&A] [answered]
I have Stable Diffusion setup using this guide "https://github.com/CompVis/stable-diffusion", and it works; 

now, when I try to setup this optimized version, and follow the instructions  and put "optimizedSD" in the same root folder with the original setup, and try to call "python optimizedSD/optimized_txt2img.py" I get the error below

![image](https://user-images.githubusercontent.com/77959003/188775762-f15357a5-6b10-447b-86fa-2b97082a3853.png)

Can anyone shed some light? I tried to fix it but can't make things work.

Thank you.

**Comment by tradlmaier** [comment, marked as answer]
If you check your paths, it is because the script leaves your second git repository(stable-diffusion-main), and enters the first one again(stable-diffusion), where the optimizedSD folder does not exist. Laziest fix would be to remove the stable-diffusion repository and rename your stable-diffusion-main.

**Reply by LeandroLosaria** [reply]
oh yes! thank you, it does make sense!
